samples/javascript_nodejs/49.WebChat-bot/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+<title>Web Chat: Full-featured bundle</title>
```

Comment:
Indentation.

HTML use 2-space indentation nowaday.

Reply:
Fixed.

BTW: Someone needs to tell the VS Code team.  The formatting was done using Ctrl+k,Ctrl+f.

Reply:
Then, please format it manually by hand.

Reply:
Already did, will be checking in soon.